pulp_ansible/app/galaxy/v3/pagination.py

```diff
@@ -10,7 +10,7 @@ class LimitOffsetPagination(pagination.LimitOffsetPagination):
     """
 
     default_limit = 10
-    max_limit = 100
+    max_limit = 1000
```

Comment:
Does this imply that OptimizedLimitOffsetPagination won't be used now? If so, presumably https://github.com/ansible/galaxy_ng/pull/726/files#diff-e43be673099bbec9ffef097ed5503589ff4597e387c89ed3ea2fdf9365e3d05dR81 needs updating as well

Reply:
if this PR is accepted then 726 can be closed without merging.